build/variables.yml

```diff
@@ -41,4 +41,6 @@ variables:
 - name: Codeql.TSAEnabled
```

Comment:
You can delete these CodeQL variables. 

I didn't check, but I'd also recommend finding the mqtt codeql pipeline and disabling it if you haven't done soe

Reply:
Yes, I've disabled the CodeQL pipeline.  And I've removed these variables.